Update devInternalSettings

alanjhughes opened the pull request:
## Summary:
The `ReactAndroidCodeTransformer` did not make the necessary modifications during the RN upgrade.

Kudo reviewed (approved):
how do you think to fix the problem by updating ReactAndroidCodeTransformer?
we can do that after beta either.

alanjhughes commented:
@Kudo I've fixed it and added it to my android PR for the expo go migration. `JavaParser` is not really suitable for modifying kotlin code and RN will continue migrating classes to it so we so we should look for better solutions for doing this.

Kudo commented:
> @Kudo I've fixed it and added it to my android PR for the expo go migration. `JavaParser` is not really suitable for modifying kotlin code and RN will continue migrating classes to it so we so we should look for better solutions for doing this.

makes sense. in this case, maybe we can edit in-place and drop the ReactAndroidCodeTransformer. let's discuss this in sdk sync.
